src/defaultSettings.js

```diff
@@ -1,5 +1,6 @@
-const defaultSettings = {
-    //Custom settings for this template
+import merge from './util/merge';
+
```

Comment:
Not sure I totally see the point of splitting the settings here, but I guess it doesn't hurt anything ... so 🤷‍♂️

Reply:
I needed to pull in the webcharts settings for the [configuration wiki](https://github.com/RhoInc/safety-results-over-time/wiki/Configuration) and this was the hacky route I chose.